articles/aks/azure-files-csi.md

```diff
@@ -227,6 +227,21 @@ az provider register --namespace Microsoft.Storage
 - secure transfer required(enable HTTPS traffic only): false
 - select the virtual network of your agent nodes in Firewalls and virtual networks
 
+### Install the azurefile CSI driver on the cluster
+
+```console
+curl -skSL https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/v0.9.0/deploy/install-driver.sh | bash -s v0.9.0 --
```

Comment:
This is not needed. This is already a cluster with CSI drivers. 
See https://docs.microsoft.com/en-us/azure/aks/csi-storage-drivers

Reply:
Interesting. I will double check on this. On a brand new 1.18.8 cluster my PVC is stuck in pending and I'm getting this on my events:

```
waiting for a volume to be created, either by external provisioner "file.csi.azure.com" or manually created by system administrator
```

this does go away once I manually install the drivers though.

Reply:
You need to enable the CSI preview feature when you create the cluster, following the link I shared. That is a pre-req for both the csi-disk and csi-file docs

Reply:
doh! you're right. I forgot to register for the 'EnableAzureDiskFileCSIDriver' :)